bin/logstashScripts/logstash.conf

```diff
@@ -1,26 +1,18 @@
 input {
     file {
-        path => "{ELK_SRC_HOME}**/*.java"
+	path => "/home/alan/gitfolder/**/*.java"
```

Comment:
?

Reply:
forgot to revert to the placeholder after testing, has been changed back